sdk/formrecognizer/ai-form-recognizer/test/public/node/analysis.spec.ts

```diff
@@ -873,23 +959,23 @@ matrix([[/* true, */ false]] as const, async (useAad) => {
           name: "BONNIE F HERNANDEZ",
           address: {
             houseNumber: "96541",
-            road: "molly hollow street",
-            city: "kathrynmouth",
-            state: "ne",
-            postalCode: "98631-5293",
-            streetAddress: "96541 molly hollow street",
+            road: "MOLLY HOLLOW STREET",
```

Comment:
All caps.. interesting

Reply:
Is this a mentionable breaking change from the previous version?

Reply:
Don't think it's breaking. The nature of the extraction is such that you just use whatever string value comes out of it, and all caps is more normal for handling address data anyways.